docs/contributing.md

```diff
@@ -211,6 +213,8 @@ Now for the process itself:
 
    Once you've made the requisite changes to the YAML file, save it and re-run `yarn create-release-branch`. You may need to repeat this step multiple times until you don't see any more errors.
 
+<a id="review-changelogs"></a>
+
```

Comment:
Is this line addition intentional?
```suggestion
```

Reply:
yes it's used for linking here https://github.com/MetaMask/core/pull/5467/files#diff-419d79ae7d440d49b3fa5fa7739f5f3a359e3ca42c3fa942e5604ae3474eb35bR293